v3/src/models/formula/functions/operators.ts

```diff
@@ -140,6 +140,11 @@ export const operators = {
       const [isANumber, aNumber] = checkNumber(a)
       const [isBNumber, bNumber] = checkNumber(b)
 
+      // Empty strings
+      if (a === "" || b === "") {
+        return ""
+      }
```

Comment:
Move this up above the `checkDate` and `checkNumber` lines. There's no reason to check those things if we're just going to bail. Also, maybe we should make the check broader just in case:
```suggestion
      // Empty strings
      if (a == null || a === "" || b == null || b === "") {
        return ""
      }
```